### ISSUE TITLE
update lib sass to version 3.3.6

### ISSUE THREAD

**Issue by dwt** [dahlia/libsass-python#154]
A new version is out, wouldn't it be great? :)

By the way: would you consider to synchronize your versions to the libsass project you release for? That way one could instantly tell which sass version was installed, without having to maintain a mapping somewhere.

**Comment by asottile** [comment]
3.3.6 is broken: https://github.com/dahlia/libsass-python/pull/149

We version differently than upstream because we add features and fix bugs and don't want to be stuck to their release cycle.

In newer versions you can retrieve the backing version in the following ways:

```
$ sassc --version
sassc 0.11.1 (sass/libsass 3.3.5)
```

or

```
$ python -c 'import sass; print(sass.libsass_version)'
3.3.5
```

**Comment by asottile** [comment]
This is now available as 0.11.2 :tada:
